Makefile.include

```diff
@@ -476,6 +476,8 @@ endif
 # set some settings useful for continuous integration builds
 ifeq ($(RIOT_CI_BUILD),1)
     RIOT_VERSION ?= buildtest
+    # set a dummy version number
+    RIOT_VERSION_CODE ?= RIOT_VERSION_NUM\(2042,5,23,0\)
```

Comment:
if somone changes the release cycle may(5) in 2042 may be a regular Release -> 
```suggestion
    RIOT_VERSION_CODE ?= RIOT_VERSION_NUM\(2042,99,23,0\)
```
Is that Feb 2050?

Reply:
Same goes, if we change for a semantic versioning scheme, but keep the year number for backwards compatibility. IMHO this is not really important. The main thing, is that the version in CI is static and since it is not planned to use this macro in-tree, it will never come to a versioning conflict within the CI.

Reply:
I think this macro might have some use inside packages that have a source that has the changes to make them work with riot upstream.
Some packages are tested in the CI ?afaik?

Reply:
If a package requires a different RIOT version, a different version of that package should be used. If we allow `RIOT_VERSION_CODE` within things the CI is using, we lose both caching advantages and size comparison capabilities offered by the CI.

Reply:
(i.e. we would loose deterministic build results within the CI)